nginx/symlinks.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+USER="${PAM_USER}"
+UID=$(id -u "$USER")
+XDG_RUNTIME_DIR="/run/user/${UID}"
+NGINX_SOCK_DIR="/run/http"
+
+for proto in http https; do
+    SOCKET_LINK="${NGINX_SOCK_DIR}/${USER}-${proto}.sock"
+    [ -L "${SOCKET_LINK}" ] || \
+	ln -s "${XDG_RUNTIME_DIR}/${proto}.sock" "${SOCKET_LINK}"
```

Comment:
Ideally, 0700 + the following ACL: `g:nginx:rwx`, and the default daemon should do that, but users can implement whatever they want.
Truthfully, arbitrary users being able to send stuff to it wouldn't seem like a big deal, given that the socket is (by nature) exposed to the whole Internet anyhow.